src/main/java/dinkplugin/DinkPluginConfig.java

```diff
@@ -612,7 +612,7 @@ default boolean clueShowItems() {
     @ConfigItem(
         keyName = "clueMinValue",
         name = "Min Value",
-        description = "The minimum value of the items to be shown",
+        description = "The minimum value of the items for a notification to be sent",
```

Comment:
Same question as #98 - should we specify that this checks the value of all items combined?